core/templates/dev/head/components/ExplorationSummaryTileDirective.js

```diff
@@ -18,6 +18,19 @@
  * @author Sean Lip
  */
 
+oppia.animation('.oppia-exploration-summary-tile-all-contributors-animate',
+  function() {
+    return {
+      enter: function(element) {
```

Comment:
I added a console.log call here and in the 'leave' function. The animation issues occur if you place the mouse cursor slightly below and to the right of the +n. I noticed that doing this fires an 'enter' event and a 'leave' event, which leads to the animation jerking up and down.

Here's a good way to debug this: add a "border: 1px solid red;" to the exploration-summary-contributors div. Do the same for contributors-number, but with a green border; do the same for all-contributors, but with a blue border. Reload the gallery page, and you'll then see clearly why the issue arises: 

![bounding-boxes](https://cloud.githubusercontent.com/assets/10575562/13196988/719ee534-d794-11e5-89aa-a8a8fe425e99.png)

The green box 'sticks out' of the right side of the card, and moving the mouse within it triggers the animation. There are also some issues with the height of the red box changing based on the appearance and disappearance of the +n. This should help with tracing the problem.